src/ansys/simai/core/data/model_configuration.py

```diff
@@ -237,6 +250,9 @@ class ModelConfiguration:
                 surface=["Pressure", "WallShearStress_0"], volume=["Velocity_0", "Pressure"]
             )
 
+            # Define the surface post-processing input
+            surface_pp_input = PostProcessInput(surface=["Temperature_1"])
```

Comment:
On one hand `PostProcessInput` is designed so it can receive other fields than `surface`, but on the other hand, the argument is called `surface_pp_input` (and eg not `pp_input`).

Reply:
arg renamed to `pp_input`; field `surface_pp_input` remains in raw model config dict!